Sources/HttpPipeline/ResponseHeader.swift

```diff
@@ -1,14 +1,87 @@
 import MediaType
+import Prelude
 
 public enum ResponseHeader {
   case allow([Method])
   case contentLength(Int)
   case contentType(MediaType)
   case location(String)
-  case setCookie([String: String])
+  case setCookie(key: String, value: String, options: Set<CookieOption>)
```

Comment:
turns out it's not legit to set multiple cookies in one go, so had to change this up. also added support for cookie options like `HttpOnly`, `Max-Age`, etc...